CMakeLists.txt

```diff
@@ -0,0 +1,222 @@
+# Copyright (C) 2025 Yubico AB - See COPYING
+
+cmake_minimum_required(VERSION 3.18.0)
+
+project(
+	pam_u2f
+	VERSION 1.3.3
+	HOMEPAGE_URL https://developers.yubico.com/pam-u2f/
+	LANGUAGES C
+)
+
+set(PROJECT_BUGREPORT https://github.com/Yubico/pam-u2f/issues)
+
+find_package(PkgConfig REQUIRED)
+include(CMakePushCheckState)
+include(CheckCCompilerFlag)
+include(CheckIncludeFile)
+include(CheckSymbolExists)
+include(CTest)
+include(GNUInstallDirs)
+
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
```

Comment:
not really a good idea, so no `CMAKE_MODULE_PATH` can be supplied from a command line call.
better use `list(APPEND`

Reply:
Hi, @akallabeth .

Thanks for your feedback.

You are technically correct (I've tried), but I don't know why the path should be supplied from command line in the first place. Could you please elaborate on your needs?

Reply:
cross compilation or non default locations of dependencies come to mind, so you have (one less) way to provide the appropriate search paths. (the `CMAKE_PREFIX_PATH` can be used as well)

Reply:
> cross compilation or non default locations of dependencies come to mind, so you have (one less) way to provide the appropriate search paths. (the `CMAKE_PREFIX_PATH` can be used as well) [edit] not had this issue with pam-u2f yet, but encountered it with other projects

I understand. Fixed in #351